tests/unittest.cpp

```diff
@@ -2472,7 +2472,7 @@ TEST_CASE("stream_response")
 
 TEST_CASE("websocket")
 {
-    static std::string http_message = "GET /ws HTTP/1.1\r\nConnection: keep-alive, Upgrade\r\nupgrade: websocket\r\nSec-WebSocket-Key: dGhlIHNhbXBsZSBub25jZQ==\r\nSec-WebSocket-Version: 13\r\n\r\n";
```

Comment:
Maybe you could copy one of this examples, but without `Host` header, to make sure that it works as expected in that case? UT which checks that error was actually fixed ;)